ci/continuous-builder.sh

```diff
@@ -87,6 +87,9 @@ if [ -n "$HASHES" ]; then
       # Setup environment. Skip build if the .env file doesn't exist any more.
       source_env_files "$env_name" || exit
 
+	  # Pass the versions down for logging
```

Comment:
Is the indentation right?

Reply:
I was using tabs by mistake, should be fixed now